### DISCUSSION TITLE
Is there any way to turn off reporting in gauge ?

### DISCUSSION THREAD

**Discussion by ayankumaar** [getgauge/gauge#2069] [General]
Hi,
I want to completely turn off reporting in gauge and I am expecting **gauge will not automatically download** the **html-plugin** and the **screenshot plugin** when I run dotnet build for the first time in my project.

What should I do to achieve this ?

I am running the following version of gauge:

Gauge version: 1.2.1

Plugins
-------
dotnet (0.4.1)

**Comment by sriv** [comment]
No, unfortunately not at the moment. Gauge tries to install `html-report` and `screenshot` by default.

I can see the need for this, please create an issue (or even a pull request) to include a config to turn this off.
